README.md

```diff
@@ -71,20 +71,20 @@ $nw-font-path: '~nw-style-guide/assets/fonts';
 ### Features and fixes
 
 1. Create a new feature branch off master
-1. Implement feature / fix in feature branch
-1. Create pull request
-1. Build the Github Pages demo in the feature branch
+2. Implement feature / fix in feature branch
+3. Create pull request
+4. Build the Github Pages demo in the feature branch
     - `npm run ghpages`
     - Make sure to `git add --all` not to end up with untracked files
     - Commit with message `build:ghpages`
     - Push changes
 
-1. Update the package version with the command `npm version {{version_type}}`. The version bump is automatically committed. See [here](https://docs.npmjs.com/cli/v6/commands/npm-version#synopsis) for further information on the `npm version` command
-1. Push changes
-1. PR is approved
-1. Merge pull request to master
-1. After merging to master, checkout master and pull the latest changes
-1. Publish the new package version to npm with `npm run package:release`
+5. Update the package version with the command `npm version {{version_type}}`. The version bump is automatically committed. See [here](https://docs.npmjs.com/cli/v6/commands/npm-version#synopsis) for further information on the `npm version` command
+6. Push changes
+7. PR is approved
+8. Merge pull request to master
+9. After merging to master, checkout master and pull the latest changes
+10. Publish the new package version to npm with `npm run package:release`
```

Comment:
I wouldn't update this as it makes adding steps to the ordered list a bit painful. If you add a step between 5 and 6 you need then update all the number after this. With markdown you can just use a single number and it will be automatically incremented. There's a few different articles out there detailing this but [here's](https://carpentries-incubator.github.io/markdown-intro/06-lists/index.html) one